tortoise.py

```diff
@@ -0,0 +1,9 @@
+def main():
+    with open('tiny.tortoise', 'r') as tiny_tortoise:
+        tortoise_program = tiny_tortoise.readlines()
+        for line in tortoise_program:
+            print(line.strip())
```

Comment:
This doesn't do what's specified [in the issue.](https://github.com/the-riviera-kid/tortoise-lang/issues/2) If you check, each line is prefixed with its line number.